terraform/testcases/otlp_metric_amp_awsprw/otconfig.tpl

```diff
@@ -0,0 +1,32 @@
+extensions:
+  pprof:
+    endpoint: 0.0.0.0:1777
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:${grpc_port}
+
+processors:
+  batch:
+
+exporters:
+  logging:
+    loglevel: debug
+  awsprometheusremotewrite:
+    endpoint: ${cortex_instance_endpoint}/api/v1/remote_write
+    aws_auth:
+      region: ${region}
+      service: "aps"
+    timeout: 10s
```

Comment:
Do we need `''` ticks?
```suggestion
  awsprometheusremotewrite:
    endpoint: '${cortex_instance_endpoint}/api/v1/remote_write'
    aws_auth:
      region: '${region}'
      service: "aps"
    timeout: 10s
```

Reply:
We don't use ticks, here or above/below; any of the tests with the `_awsprw` suffix are the same as tests that already exist in the repository (and thus have been passing already)